packages/template-chakra-storefront/src/pages/product-list/hooks/use-product-list-data.js

```diff
@@ -115,7 +114,7 @@ export const useProductListData = () => {
         filtersLoading,
         handleProductClick,
         isFetched,
-        isLoading,
+        isPending,
```

Comment:
Have you thought about aliasing isPending as isLoading so that we don't have to make knock on changes throughout the codebase where this hook is used. 

I can see the positives to doing either, but I want to get your thoughts on it.

Reply:
I think it is better to use `isPending` as it is from react query. This is `isLoading` is still a legit prop from react query v5. See [here](https://tanstack.com/query/v5/docs/framework/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading). `isLoading` was `IsInitialLoading` . If we use alias here, we are removing `isLoading` from react-query entirely